src/radiation_system.hpp

```diff
@@ -1108,7 +1101,7 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 			quokka::valarray<double, nGroups_> deltaErad;
 			quokka::valarray<double, nGroups_> F_R;
 
-			const double resid_tol = 1.0e-13; // 1.0e-15;
+			const double resid_tol = 1.0e-10; // 1.0e-15;
```

Comment:
What is the motivation for this change? This can be made a runtime parameter if it needs to be adjusted per-problem.

Reply:
I just found no benifit is gain from setting a lower resid_tol. Besides, the variable-Kappa advecting pulse problem failed to converged, and I changed this back while trying to resolve it, although it didn't help. I think 1e-10 is more than enough for good conservation.